conf/nginx.conf

```diff
@@ -108,7 +108,7 @@ http {
             allow 127.0.0.0/24;
             deny all;
 
-            root ../;
+            alias dashboard/;
```

Comment:
is `alias dashboard;` better? I think we can omit the last `/`

Reply:
Yeah!  But I prefer change the location prefix to `/apisix/dashboard/` and keep slash in the `alias`, so the /apisix/dashboardxxx wouldn't match this location.

Reply:
Same reason, I also prefer the `/apisix/admin/` than `/apisix/admin`, so what's you opinion?

Reply:
I think `location /apisix/dashboard` is the same as `location /apisix/dashboard/` in nginx.conf.

Reply:
Not the same.
``` nginx
        location /apisix/dashboard {
            return 302 https://www.baidu.com;
        }

        location /apisix/dashboard/ {
            index index.html;
            allow 127.0.0.0/24;
            allow 122.224.174.110;
            deny all;

            alias dashboard/;

            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Real-PORT $remote_port;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;

            try_files $uri $uri/ /index.html;
        }
```
```
$ curl -i http://127.0.0.1:9080/apisix/dashboard
HTTP/1.1 302 Moved Temporarily
Date: Thu, 08 Aug 2019 03:13:32 GMT
Content-Type: text/html
Content-Length: 142
Connection: keep-alive
Location: https://www.baidu.com
Server: APISIX web server

<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

$ curl -i http://127.0.0.1:9080/apisix/dashboardxyz
HTTP/1.1 302 Moved Temporarily
Date: Thu, 08 Aug 2019 03:14:35 GMT
Content-Type: text/html
Content-Length: 142
Connection: keep-alive
Location: https://www.baidu.com
Server: APISIX web server

<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>openresty</center>
</body>
</html>

$ curl -i http://127.0.0.1:9080/apisix/dashboard/
HTTP/1.1 200 OK
Date: Thu, 08 Aug 2019 03:14:02 GMT
Content-Type: text/html
Content-Length: 4299
Last-Modified: Wed, 07 Aug 2019 08:42:36 GMT
Connection: keep-alive
ETag: "5d4a8efc-10cb"
Server: APISIX web server
Accept-Ranges: bytes

<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><!--[if IE]><link rel="icon" href="/apisix/dashboard/favicon.ico"><![endif]--><title>Dashboard for APISIX</title><link href=/apisix/dashboard/css/401.3f124f64.css rel=prefetch><link href=/apisix/dashboard/css/404.ad9526b5.css rel=prefetch><link href=/apisix/dashboard/css/chunk-10ba0bbe.4cc2e8dd.css rel=prefetch><link href=/apisix/dashboard/css/chunk-14349a2e.8633aaf3.css rel=prefetch><link href=/apisix/dashboard/css/chunk-253d8d80.7a4df7e9.css rel=prefetch><link href=/apisix/dashboard/css/chunk-31d45238.75e54251.css rel=prefetch><link href=/apisix/dashboard/css/chunk-3891d839.94c1ba11.css
```

Reply:
ok, you are right:) 
please modify https://github.com/iresty/apisix/blob/master/bin/apisix#L201 too.

Reply:
OK